src/app/shared/module/poe/component/item-frame-properties-ultimatum/item-frame-properties-ultimatum.component.scss

```diff
@@ -0,0 +1,9 @@
+@import './../../../../../../styles/variables';
+
+.white {
+  color: $white;
+}
+
+.purpel {
```

Comment:
spelling if you care

Reply:
It's incorrect all over the place. Might fix it for the entire project somewhere in the future.